demisto_sdk/commands/common/errors.py

```diff
@@ -1162,6 +1197,28 @@ def integration_not_runnable():
         return "Could not find any runnable command in the integration." \
                "Must have at least one command, `isFetch: true`, `feed: true`, `longRunning: true`"
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_entity_description_no_escape_sequences():
+        return 'The entity description can not contain Escape Sequences (\\n \\t \\r etc.).'
+
+    @staticmethod
+    @error_code_decorator
+    def invalid_command_description_no_escape_sequences(invalid_commands_list, invalid_arguments_list,
+                                                        invalid_outputs_list):
+        error_msg = 'The command description can not contain Escape Sequences (\\n \\t \\r etc.), ' \
+                    'please remove the escape sequences form'
+        if invalid_commands_list:
+            error_msg += f' commands description: {invalid_commands_list}.'
+
+        if invalid_arguments_list:
+            error_msg += f' args description: {invalid_arguments_list}.'
+
+        if invalid_outputs_list:
+            error_msg += f' outputs description: {invalid_outputs_list}.'
+
+        return error_msg
```

Comment:
I think this will look too messy we should reorder this - I think it might be wiser to separate this into 3 different errors and print it separately.

Reply:
done